assets/components/digitalSubscriptions/digitalSubscriptions.jsx

```diff
@@ -0,0 +1,208 @@
+
+import PageSection from 'components/pageSection/pageSection';
+import SubscriptionBundle from 'components/subscriptionBundle/subscriptionBundle';
+import { type HeadingSize } from 'components/heading/heading';
+
+
+// ----- Types ----- //
+
+type ClickEvent = () => void;
+
+type PropTypes = {
+  referrerAcquisitionData: ReferrerAcquisitionData,
+  headingSize: HeadingSize,
+  clickEvents: {
+    iOSApp: ClickEvent,
+    androidApp: ClickEvent,
+    dailyEdition: ClickEvent,
+    digiPack: ClickEvent,
+  },
+};
+
+
+// ----- Setup ----- //
+
+const gridImageProperties = {
+  srcSizes: [825, 500, 140],
+  sizes: '(max-width: 660px) 165px, (max-width: 740px) 174px, (max-width: 980px) 196px, (max-width: 1140px) 205px, 165px',
+  imgType: 'png',
+};
+
+const appReferrer = 'utm_source=support.theguardian.com&utm_medium=subscribe_landing_page&utm_campaign=split_subscriptions_test';
+
+
+// ----- Component ----- //
+
+export default function DigitalSubscriptions(props: PropTypes) {
+
+  const subsLinks = getSubsLinks(
+    props.referrerAcquisitionData.campaignCode,
+    getCampaign(props.referrerAcquisitionData),
+    [],
+    props.referrerAcquisitionData,
+  );
+
+  return (
+    <div className="component-digital-subscriptions">
+      <PageSection
+        heading="Digital Subscriptions"
+        modifierClass="digital-subscriptions"
+      >
+        <PremiumTier
+          iOSUrl={addQueryParamsToURL(iOSAppUrl, { referrer: appReferrer })}
+          androidUrl={addQueryParamsToURL(androidAppUrl, { referrer: appReferrer })}
+          headingSize={props.headingSize}
+          iOSOnClick={props.clickEvents.iOSApp}
+          androidOnClick={props.clickEvents.androidApp}
+        />
+        <DailyEdition
+          url={addQueryParamsToURL(dailyEditionUrl, { referrer: appReferrer })}
+          headingSize={props.headingSize}
+          onClick={props.clickEvents.dailyEdition}
+        />
+        <DigitalBundle
+          url={subsLinks.digital}
+          headingSize={props.headingSize}
+          onClick={props.clickEvents.digiPack}
+        />
+      </PageSection>
+    </div>
+  );
+
+}
+
+
+// ----- Auxiliary Components ----- //
+
+function PremiumTier(props: {
+    iOSUrl: string,
+    androidUrl: string,
+    headingSize: HeadingSize,
+    iOSOnClick: ClickEvent,
+    androidOnClick: ClickEvent,
+}) {
+
+  return (
+    <SubscriptionBundle
+      modifierClass="premium-tier"
+      heading="Premium App"
+      subheading="£5.99/month"
+      headingSize={props.headingSize}
+      benefits={[
+        {
+          text: 'The ad free, premium app, designed especially for your smartphone and tablet',
+        },
+      ]}
+      gridImage={{
+        gridId: 'premiumTierCircle',
+        altText: 'premium app',
+        ...gridImageProperties,
+      }}
+      ctas={[
+        {
+          text: 'Buy in the App Store',
+          url: props.iOSUrl,
+          accessibilityHint: 'Proceed to buy the premium app in the app store',
```

Comment:
Proceed to buy and download the premium app from the Apple app store?

Reply:
This whole text is slightly misleading because all you can do in the app store is download the app, you have to buy the premium tier in app once it's installed.